.github/workflows/build_and_test.yml

```diff
@@ -277,6 +278,179 @@ jobs:
           path: ./tests/results/${{ matrix.test-suite.name }}/output.xml
           if-no-files-found: error
 
+  #
+  # Uses the ehrbase docker image from [build] to run the performance tests against it.
+  #
+  performance-test-run:
+    runs-on: ubuntu-latest
+    needs: [
+      docker-test-image
+    ]
+    name: Perf (${{ matrix.test-plan.name }})
+    strategy:
+      fail-fast: false # ensure all tests run
+      matrix:
+        test-plan: [
+          { name: 'ehrbase_no_auth_scenario', env: { 'AUTH_TYPE': 'NONE' } },
+          { name: 'ehrbase_basic_auth_scenario', env: { 'AUTH_TYPE': 'BASIC' } },
+          { name: 'ehrbase_oauth_auth_scenario', env: { 'AUTH_TYPE': 'OAUTH' } },
+        ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            tests
+            .env.ehrbase
+            docker-compose.yml
+
+      - name: Download - Test Image
+        uses: actions/download-artifact@v4
+        with:
+          name: ehrbase-image-test
+          path: ${{ runner.temp }}
+
+      - name: Docker - Load Image
+        run: docker load --input ${{ runner.temp }}/ehrbase-test.tar
```

Comment:
I'm not entirely sure but possible we should upload and use the `ehrbase/ehrbase:build` image from here https://github.com/ehrbase/ehrbase/blob/feature/CDR-1325_perf_tests_pipeline/.github/workflows/build_and_test.yml#L120 ... the `ehrbase/ehrbase:test` image run with jacococo coverage agent that "could" have some negative performance impact.

Reply:
fixed